### PR TITLE
LP-2347 ADG header integrated

### PR DESCRIPTION
**Pull request by annas-usman**
constant for CDN link has been set in lms/envs/common.py file

Adjacent pull request in adg-edx-theme
[24](https://github.com/OmnipreneurshipAcademy/adg-edx-theme/pull/24)

[Ticket](https://philanthropyu.atlassian.net/browse/LP-2347)

### CODE REVIEW

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/432/workflows/workflows_id/jobs/432/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
No lines with coverage information in this diff.
-------------
```

**Review by zahid-arbisoft — APPROVED** [review]

File: lms/envs/common.py [hunk 1]
```diff
@@ -3933,3 +3933,6 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
 # Notification email settings
 NOTIFICATION_FROM_EMAIL = 'info@omnipreneurshipacademy.com'
+
+# CDN link
```
Comment (harris-irfan) [inline lms/envs/common.py:3937]: Can we better describe this, like what is this CDN link used for, the way it is described in line 487?

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/457/workflows/workflows_id/jobs/457/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
No lines with coverage information in this diff.
-------------
```

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/460/workflows/workflows_id/jobs/460/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
No lines with coverage information in this diff.
-------------
```

**Review by harris-irfan — APPROVED** [review]

**Pull request merged**